geppetto.js/package.json

```diff
@@ -0,0 +1,186 @@
+{
```

Comment:
Remove it, I will create a new slim version for the tests but otherwise each package has its own package.json